week1-section3/exercise3/Main.cpp

```diff
@@ -0,0 +1,83 @@
+/*Exercise 2 - More Input Validation
```

Comment:
Same comment about naming your projects. This is exercise 2 but the project is called exercise3.